modules/ROOT/pages/cloudhub-setup.adoc

```diff
@@ -54,6 +54,15 @@ To enable the Anypoint Monitoring agent:
 
 For more information about Anypoint Monitoring, see xref:monitoring::enable-apps-deployed-to-cloud.adoc[Enable Monitoring Using the UI].
 
+==Verify Supported Mule Runtime Engine (Mule) Versions for Policies
```

Comment:
Looks like the ascii is still here (== shows up rather than this as a title)

Reply:
```suggestion
== Verify Supported Mule Runtime Engine (Mule) Versions for Policies
```